inputCandidateGenerator.js

```diff
@@ -42,6 +42,9 @@ function sTreeGEN(terminalString, options)
     var sTreeList = autoSTreePairs.map(x=>x[1]);
 
     //Apply filters
+    if(options.noBarLevels){
+      options.maxBranching = 3;
```

Comment:
I was thinking we would set this in interface1.js, rather than adding noBarLevels as an option to sTreeGEN() itself.